### PR TITLE
fix: update RULES.md with missing rule

### LINKED ISSUE #1329: Update RULES.md with after PR1321
### Describe the problem

A new rule has been added in #1321 and [RULES.md](https://github.com/MobilityData/gtfs-validator/blob/master/RULES.md) needs to be updated.

### Suggest a fix or an alternative

_No response_

### PR DESCRIPTION
**Pull request by briandonahue**
**Summary:**

Closes #1329. Missing rule definition from PR #1321 

**Expected behavior:** 

Rule should have been added to RULES.md

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Run the unit tests with `gradle test` to make sure you didn't break anything
- [x] Format the title like "feat: [new feature short description]". Title must follow the Conventional Commit Specification(https://www.conventionalcommits.org/en/v1.0.0/).
- [x] Linked all relevant issues
- [ ] Include screenshot(s) showing how this pull request works and fixes the issue(s)

### CODE REVIEW

File: RULES.md [hunk 1]
```diff
@@ -81,6 +81,7 @@ Each Notice is associated with a severity: `INFO`, `WARNING`, `ERROR`.
...
 | [`stop_time_with_arrival_before_previous_departure_time`](#stop_time_with_arrival_before_previous_departure_time)                 | Backwards time travel between stops in `stop_times.txt`                                                                                                |
 | [`stop_time_with_only_arrival_or_departure_time`](#stop_time_with_only_arrival_or_departure_time)                                 | Missing `stop_times.arrival_time` or `stop_times.departure_time`.                                                                                      |
+| [`stop_without_location`](#stop_without_location)                                                                                   | `stop_lat` and/or `stop_lon` missing for strop with `location_type` of `0`, `1`, or `2`.                                                                                                                |
```
Comment (bdferris-v2) [inline RULES.md:84]: nit: is the table alignment off here?

File: RULES.md [hunk 2]
```diff
@@ -1408,6 +1409,28 @@ Missing `stop_time.arrival_time` or `stop_time.departure_time`
...
+
+#### References
+* [GTFS stops.txt specification](https://gtfs.org/reference/static#stopstxt)
+<details>
+
+#### Notice fields description
+| Field name               	| Description                                	| Type   	|
+|--------------------------	|--------------------------------------------	|--------	|
+| `stopId`                 	| The faulty record's id.                    	| String 	|
+| `locationType`            | The faulty record's `stops.location_type`. 	| Integer 	|
```
Comment (bdferris-v2) [inline RULES.md:1426]: nit: ditto on alignment

File: RULES.md (hunk 1 above)
Comment (briandonahue) [reply RULES.md:84]: It actually doesn't look off to me. The lines are so wide, you have to stretch the editor window full screen to see alignment.  Here's the 10,000 foot view:
image

File: RULES.md [hunk 3]
```diff
@@ -81,6 +81,7 @@ Each Notice is associated with a severity: `INFO`, `WARNING`, `ERROR`.
 | [`stop_time_timepoint_without_times`](#stop_time_timepoint_without_times)                                                         | `arrival_time` or `departure_time` not specified for timepoint.                                                                                        |
 | [`stop_time_with_arrival_before_previous_departure_time`](#stop_time_with_arrival_before_previous_departure_time)                 | Backwards time travel between stops in `stop_times.txt`                                                                                                |
 | [`stop_time_with_only_arrival_or_departure_time`](#stop_time_with_only_arrival_or_departure_time)                                 | Missing `stop_times.arrival_time` or `stop_times.departure_time`.                                                                                      |
+| [`stop_without_location`](#stop_without_location)                                                                                 | `stop_lat` and/or `stop_lon` missing for strop with `location_type` of `0`, `1`, or `2`                                                                |
```
Comment (isabelle-dr) [inline RULES.md:84]:
```suggestion
| [`stop_without_location`](#stop_without_location) | `stop_lat` and/or `stop_lon` is missing for stop with `location_type` equal to`0`, `1`, or `2`
```

File: RULES.md [hunk 4]
```diff
@@ -1408,6 +1409,28 @@ Missing `stop_time.arrival_time` or `stop_time.departure_time`
 
 </details>
 
+<a name="StopWithoutLocationNotice"/>
+
+### stop_without_location
+
+`stop_lat` and/or `stop_lon` missing for strop with `location_type` of `0`, `1`, or `2`.
```
Comment (isabelle-dr) [inline RULES.md:1416]:
```suggestion
`stop_lat` and/or `stop_lon` are required for locations that are stops (`location_type=0`), stations (`location_type=1`) or entrances/exits (`location_type=2`).
```

**Review by isabelle-dr — APPROVED** [review]
Minor changes in wording, LGTM!

**Review by fredericsimard — APPROVED** [review]

**Comment by bdferris-v2** [comment]
@fredericsimard I don't think it's enough for you to approve this PR.  I believe you have to be the one to merge it as well.

**Comment by fredericsimard** [comment]
@bdferris-v2 I didn't know what the last step was, hence why I didn't merge it:

> - [ ] Include screenshot(s) showing how this pull request works and fixes the issue(s)

**Pull request merged**